TwitchDownloaderWPF/Services/AvailableCultures.cs

```diff
@@ -19,15 +19,16 @@ public static class AvailableCultures
         // ISO 3166-1 alpha-2: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements
 
         public static readonly Culture English;
-        public static readonly Culture Spanish;
         public static readonly Culture French;
         public static readonly Culture Italian;
         public static readonly Culture Japanese;
         public static readonly Culture Polish;
         public static readonly Culture Russian;
+        public static readonly Culture Spanish;
```

Comment:
Why did you change the position of Spanish? It was ordered based on the native names, not the English names.